community/github/pull-requests.md

```diff
@@ -0,0 +1,211 @@
+# 2. pyOpenSci pull request process
+
+When possible, pull request and issue submissions and reviews should follow standard open source workflows. Below are guidelines for pull requests.
+
+## Pull Requests
+
+New pull requests should:
+
+* be created from a fork rather than the pyOpenSci owned parent repository.
```

Comment:
another note on consistency: sometimes the first word in a bullet is capitalized, but other times is lowercase. MLA style is to keep it lowercase unless it's a proper noun.